alcs-frontend/src/app/features/application/applicant-info/application-details/parcel/parcel.component.html

```diff
@@ -40,6 +40,10 @@ <h4>Parcel {{ parcelInd + 1 }}: Parcel and Owner Information</h4>
     <div class="grid-double">
       {{ parcel.purchasedDate | date }}
     </div>
+    <div class="subheading2 grid-1">Civic Address</div>
+    <div class="grid-double">
```

Comment:
is there an updated flag for civic address?

Reply:
Done! Resolved conflicts and added update banner for Civic Address